kcards/views/api_rooms.py

```diff
@@ -11,7 +11,8 @@
 def index():
     rooms = sorted(Room.objects)
 
-    content = [url_for('.detail', code=r.code, _external=True) for r in rooms]
+    content = [url_for('api_rooms.detail', code=room.code, _external=True)
```

Comment:
Apparently these need the full blueprint name/method if they're to be called from another blueprint.